api/tx_args.go

```diff
@@ -146,7 +142,27 @@ func (args *SendTxArgs) setDefaults(ctx context.Context, b Backend) error {
 		nonce := b.GetPoolNonce(ctx, args.From)
 		args.AccountNonce = (*hexutil.Uint64)(&nonce)
 	}
-
+	// Estimate the gas usage if necessary.
+	if args.GasLimit == nil {
+		// We accept "data" and "payload(=input)" for backwards-compatibility reasons.
+		// "payload(=input)" is the newer name and should be preferred by clients.
```

Comment:
```suggestion
		// but input is preferred.
```

Reply:
Could you please elaborate the reason for this change request?